src/frontend/types.ts

```diff
@@ -55,12 +55,22 @@ export interface ContextType {
     add: (appNameToAdd: string, appTitle: string) => void
     remove: (appNameToRemove: string) => void
   }
+  customCategories: {
+    list: Record<string, string[]>
+    listCategories: () => string[]
+    add: (category: string, appName: string) => void
+    remove: (category: string, appName: string) => void
```

Comment:
```suggestion
    addToGame: (category: string, appName: string) => void
    removeFromGame: (category: string, appName: string) => void
```

so it's more consistent with the next 2 functions